docs/examples.md

```diff
@@ -78,4 +78,106 @@
 
 2. Curl it via localhost
 
-    `curl localhost:8082/`
\ No newline at end of file
+    `curl localhost:8082/`
+
+## Local insecure registry
```

Comment:
The examples have some sort of "action" character, so maybe calling it "Connect with a local insecure registry" fits better?

Reply:
Fixes